src/classes/CRLP_Batch_Base_Skew.cls

```diff
@@ -130,7 +129,8 @@ public abstract class CRLP_Batch_Base_Skew extends CRLP_Batch_Base {
                         parentRelationshipPrefix = Opportunity.fields.Primary_Contact__c.getDescribe().getName().replace('__c', '__r') + '.';
                     } else if (this.jobType == CRLP_RollupProcessingOptions.RollupType.ContactSoftCredit) {
                         parentRelationshipPrefix = 'Contact.';
-                    } else if (this.jobType == CRLP_RollupProcessingOptions.RollupType.AccountContactSoftCredit) {
+                    } else if (this.jobType == CRLP_RollupProcessingOptions.RollupType.AccountContactSoftCredit || 
+                            this.jobType == CRLP_RollupProcessingOptions.RollupType.ContactSoftCredit) {
                         parentRelationshipPrefix = 'Contact.Account.';
```

Comment:
This isn't one of those if blocks that needed changing. If you look at line 130 above this it's already checking for `(this.jobType == CRLP_RollupProcessingOptions.RollupType.ContactSoftCredit)` so the change made in line 132 will never find match the `||` condition. Plus the `parentRelationshipPrefix` for a Contact rollup should not be `'Contact.Account.'` because that points to the AccountId as opposed to the ContactId.

You can remove this change.

Reply:
@force2b that makes sense. I will update.